### PR TITLE
update clone domain to copy custom settings

### PR DESCRIPTION
**Pull request by czue**
@proteusvacuum this is why they didn't carry over.

cc @snopoke @orangejenny buddy

### CODE REVIEW

File: corehq/apps/hqadmin/management/commands/clone_domain.py [hunk 1]
```diff
@@ -317,15 +325,19 @@ def save_couch_copy(self, doc, new_domain=None):
         self.log_copy(doc.doc_type, old_id, new_id)
         return old_id, new_id
 
-    def save_sql_copy(self, model, new_domain):
-        old_id = model.id
+    def save_sql_copy(self, model, new_domain, domain_is_pk=False):
+        old_id = model.domain if domain_is_pk else model.id
```
Comment (snopoke) [inline corehq/apps/hqadmin/management/commands/clone_domain.py]: you could just change this to `model.pk` which will work for all models (except ones with composite primary keys)

File: corehq/apps/hqadmin/management/commands/clone_domain.py [hunk 2]
```diff
@@ -317,15 +325,19 @@ def save_couch_copy(self, doc, new_domain=None):
...
+
         if self.no_commmit:
-            model.id = 'new-{}'.format(old_id)
+            if not domain_is_pk:
+                model.id = 'new-{}'.format(old_id)
         else:
             model.save()
-        new_id = model.id
+
+        new_id = model.domain if domain_is_pk else model.id
```
Comment (snopoke) [inline corehq/apps/hqadmin/management/commands/clone_domain.py]: same as above

File: corehq/apps/hqadmin/management/commands/clone_domain.py (hunk 1 above)
Comment (czue) [inline corehq/apps/hqadmin/management/commands/clone_domain.py]: good call, updated

**Review by snopoke — APPROVED** [review]

**Comment by proteusvacuum** [comment]
cool! another thing I noticed that didn't carry over are the Repeaters. Not sure if there is a list of things that still need to get added to this.

**Comment by czue** [comment]
Yeah we did it opportunistically for icds so not everything is supported.
Sorry about that. Saw the ticket so can handle through there

**Pull request merged**